presentation/common/src/presentation-common/hierarchy/Key.ts

```diff
@@ -158,6 +158,9 @@ export interface BaseNodeKey {
 
   /** Node hash path from root to the node whose key this is */
   pathFromRoot: string[];
+
+  /** Query that returns all selected instance keys */
+  instanceKeysSelectQuery: InstanceKeysSelectQuery;
```

Comment:
shouldn't `instanceKeysSelectQuery` be optional?

Reply:
Made optional.